indico/modules/events/editing/controllers/backend/editable_list.py

```diff
@@ -0,0 +1,57 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2020 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from flask import jsonify, request
+
+from indico.modules.events import Event
+from indico.modules.events.contributions.models.contributions import Contribution
+from indico.modules.events.editing.controllers.base import RHEditingBase
+from indico.modules.events.editing.models.editable import EditableType
+
+
+class RHEditableList(RHEditingBase):
+    """Return editable list of the event for the editable type."""
+    def _process_args(self):
+        RHEditingBase._process_args(self)
+        self.event = Event.get(int(request.view_args['confId']))
+        self.editable_type = EditableType[request.view_args['type']]
```

Comment:
I guess you can inherit from `RHEditableTypeManagementBase` to have `self.editable_type` as well